docs/release-notes/release-1-4-0.md

```diff
@@ -0,0 +1,69 @@
+# Release Notes - ISLE v.1.3.0, 2019-09-27
+
+### Contributions to this release from:
+
+* David Keiser-Clark (Williams College), documentation
+* Gavin Morris (Born-Digital), code updates, documentation and testing
+* Mark McFate (Grinnell College), documentation, code updates and testing
+* Mark Sandford (Colgate University), documentation
+* Bethany Seeger (Amherst College), testing and release manager
+* Noah Smith (Born Digital), code updates and testing
+* Shaun Trujillo (Mount Holyoke College), documentation review
+
+### ISLE (main project)
+
+
+* This is an ISLE security release.  Images have been updated and only minor adjustments in version happened.
+
+### Upgrade
+
+If you are coming from ISLE releases `1.3.0` - there will be minamal steps to update. 
+If you are coming from an ISLE release older then `1.3.0`, please read the release notes for `1.3.0` to understand
+the paradigm shift that happened between previous releases. 
```

Comment:
the paradigm shift that happened between previous releases.  You will also need to follow the instructions given in the [Cleanup ISLE git repository for Release v.1.4.0](https://islandora-collaboration-group.github.io/ISLE/cookbook-recipes/isle-v140-git-cleanup/) documentation.

Reply:
Adding this in

Reply:
Fixed now